### ISSUE TITLE
Training data of stage 2

### ISSUE THREAD

**Issue by AAAeray** [JianghaiSCU/LightenDiffusion#4]
Hi,
Thanks for your great work, It seems that there is no training data of stage 2, it's reported as 180k unpaired data in paper but the linked data in [README]  are obviously less than 180k. Besides, does it means there are 90k lowlight+90k normal light or some other settings. And is there any clue that howdataset size affects results, do you have any sugesstion about how to collect data and the minimun dataset size.
Thanks

**Comment by JianghaiSCU** [comment]
We will release the used training data. Of course, you can train with unpaired data in random combinations of existing datasets.

**Comment by AAAeray** [comment]
Thanks for your replying, and about stage 1, there seems no training code and loss functions, how to train with it?

**Comment by 66ling66** [comment]
I trained the first stage use the author's loss function in paper,but I did not get the same results as the released weight.Can you tell me more training details about stage 1?
